### PR TITLE
Microverse Capstone Project

### PR DESCRIPTION
**Pull request by jlcarrascof**
In this branch, I worked all the week on: 

- [ ] Create a mobile version.
- [ ] Create media queries for the desktop version.
- [ ] Add items dinamically using Javascript.
- [ ] Fix Linters errors.

**OBSERVATION:**

**I will record the video after my Code Reviewers observations, to record the final version. Thanks for your comprehension.**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        <li><img src="./assets/images/Renault.png" alt="Renault"></li>
+        <li><img src="./assets/images/Rolex.png" alt="Rolex"></li>
+        <li><img src="./assets/images/Wilson.png" alt="Wilson"></li>
+      </ul>
+    </section>
+    <footer class="lato">
+      <img src="./assets/images/logo-rg.svg" class="logo" alt="Roland Garros">
+      <p>©2023 - all rights reserved. Fédération Française de Tennis <br>
+        All content on the website, unless otherwise specified, is available under a Creative Commons Attribution</p>
+    </footer>
```
Comment (MaganaAsiati) [inline about.html]: - In the Desktop Size, for the footer :+1: 
About Page: It has a dark background.
Home Page: It has a white background.


Kindly Fix that, by making   the footer for about section dark as required in the project :+1: 

## Your design 
image

## design provided 

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      <h1 class="cocogoose">ROLAND GARROS 2023 <br>
+        THE MAIN TENNIS EVENT IN THE SEASON IS COMING
+        IN MAY 2023
+      </h1>
+      <p class="lato">Be part of it all in Opening Week (22-26 May)<br>
+      Come and watch the qualifying matches, the stars on the practice courts and much more besides in Opening Week at this year’s Parisian Grand Slam event.</p>
+      <h2 class="cocogoose">22 MAY - 11 JUN 2023</h2>
+      <h4 class="lato">©2023 - all rights reserved
+        Fédération Française de Tennis</h4>
+    </section>
```
Comment (MaganaAsiati) [inline index.html:61]: - Kindly make sure that you add the appropriate  background images to this section to make it look luck the design provided as required in the project :+1: 

## Your design 

image

## The design provided 


image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+          Roland Garros is unmatched. He has won the tournament almost every year since 
+          his first win in 2005, except for a few years when he was either injured or 
+          unable to compete. Nadal’s success at the tournament has solidified his place 
+          in tennis history and has made him a household name around the world. His rivalry 
+          with Roger Federer and Novak Djokovic has also added to the tournament’s appeal, 
+          making it a highly anticipated event every year.
+      </p>
+      <p class="lato about-info">Please contact us per email for any further questions about Roland Garros Tournament!</p>
+      <h5 class="lato">info@rolandgarros.com</h5>
+    </section>
```
Comment (MaganaAsiati) [inline about.html:79]: - Kindly make sure that you add the appropriate  background images to this section to make it look luck the design provided as required in the project :+1: 

## Your design 

image

## The design provided 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @jlcarrascof 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️


- Kindly recorded a video presenting the features of the project, the maximum length of the video is 5 minutes. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear. The video contains a demo of the project, showing the pages in different screen sizes. The video contains a highlight about a piece of the code the student finds interesting or is proud of making. :+1: 

- Kindly makes use of GitHub Pages to deploy the web pages. The project is deployed properly, and all pages, links, and images work as expected. and also make sure a link to the deployed project is present in the README. :+1: 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: about.html (hunk 1 above)
Comment (jlcarrascof) [reply about.html]: issue-01

**Fixed** the footer (Desktop size) in about.html.

File: index.html (hunk 2 above)
Comment (jlcarrascof) [reply index.html:61]: issue-02

**Fixed the background image** in index.html

File: about.html (hunk 3 above)
Comment (jlcarrascof) [reply about.html:79]: issue-03

**Fixed the background image in about.html** page.

**Comment by jlcarrascof** [comment]
And last but not least, **I fixed my README.md** according these suggestions:

Required Changes ♻️

    Kindly recorded a video presenting the features of the project, the maximum length of the video is 5 minutes. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear. The video contains a demo of the project, showing the pages in different screen sizes. The video contains a highlight about a piece of the code the student finds interesting or is proud of making. 👍

    Kindly makes use of GitHub Pages to deploy the web pages. The project is deployed properly, and all pages, links, and images work as expected. and also make sure a link to the deployed project is present in the README. 👍

I mean: 

- [ ]  I **recorded** my **Loom Video**.
- [ ]  I **deployed** my updated website on **GitHub Pages**.

Thanks for your time, dear @MaganaAsiati 

Javier.-

File: README.md [hunk 4]
```diff
@@ -1,2 +1,170 @@
...
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 HTML/CSS & JavaScript capstone project - Model: Conference page <a name="about-project"></a>
+
+**First capstone project** is a web page where you can find information about Rolland Garros Tournament.<br>
+
+In [this video](https://www.loom.com/share/3592df7602c04fd7ae8531b6eca35715) you can find more information about this project.
```
Comment (amejid) [inline README.md]: - [ ] You did a great job presenting the project and implementing the required changes. :+1: I think you forgot to highlight a piece of code you feel most proud of. Please record the video again to include that detail. 🤝️

**Project requirements**
image

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @jlcarrascof, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Great UI design :heavy_check_mark: 
- Good presentation :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (jlcarrascof) [reply README.md]: **I just recorded my video again**, sir, following your kind suggestions.

**https://www.loom.com/share/4f041649025548649006fb8bf20d2b74**

**The Readme.md file was updated as well.**

Thanks, Mr. **@amejid**

**Review by emmiiorji — APPROVED** [review]
Hi @jlcarrascof, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>


## Good Highlights✔️

- Requested changes have been implemented
- Good job deploying the project

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jlcarrascof** [comment]
Thanks to my code reviewers in this project:

- [ ] **@MaganaAsiati** 
- [ ] **@amejid** 
- [ ] **@emmiiorji** 

For all their valuable feedback to fix my mistakes and all my friends in Microverse community, especially my mates from Colombia.

Javier.-
Venezuela.

**Pull request merged**